### PR TITLE
feat: reopen library menu on import from file

### LINKED ISSUE #2361: When importing to library, show the library
Because at the moment, nothing happens visually when dropping the lib file into the scene (or opening it)

### PR DESCRIPTION
**Pull request by ThangHuuVu**
resolve #2361

### CODE REVIEW

File: src/components/LayerUI.tsx [hunk 1]
```diff
@@ -298,9 +298,31 @@ const LibraryMenu = ({
...
+  };
 
   return loadingState === "preloading" ? null : (
-    <Island padding={1} ref={ref} className="layer-ui__library">
+    <Island
+      padding={1}
+      ref={ref}
+      className="layer-ui__library"
+      onDrop={handleLibraryMenuOnDrop}
+    >
```
Comment (dwelle) [inline src/components/LayerUI.tsx:325]: Actually, can we do this in a separate PR to keep the PRs self-contained (not sure why I said that in the issue). Mainly because I just realized we should maybe put the drop handler on the whole excalidraw container instead of having multiple, but it'll be a more major change so best leave it for a separate PR.

Thanks!

Comment (ThangHuuVu) [reply src/components/LayerUI.tsx:325]: Sure thing, I'll revert this change. If possible I would like to be assigned to that issue too!

**Comment by dwelle** [comment]
Looks good. Thanks @ThangHuuVu!

**Review by dwelle — APPROVED** [review]
